colossalai/tensor/colo_parameter.py

```diff
@@ -3,23 +3,23 @@
 import torch
 from colossalai.tensor import TensorSpec, distspec
 from copy import copy
+from typing import Optional
 
-
-class ColoParameter(ColoTensor):
+class ColoParameter(ColoTensor, torch.nn.Parameter):
```

Comment:
Can ColoParameter use __torch_function__?
Can you add a test for this function?

Reply:
It works.
![image](https://user-images.githubusercontent.com/18738848/170945797-8c75b36c-28c1-42ea-8724-a9a3f84fb537.png)